src/llvm/ext/llvm_ext.cc

```diff
@@ -56,14 +60,16 @@ LLVMDIBuilderRef LLVMNewDIBuilder(LLVMModuleRef mref) {
   return wrap(new DIBuilder(*m));
 }
 
+#if LLVM_VERSION_LE(5, 9)
```

Comment:
Note: there won't be any 5.x other than 5.0. `LE(5, 0)` is enough, no need to indroduce unknown version numbers :)

Reply:
Good catch, I just fixed this locally, but @RX14 merged in parallel before I pushed it.

I'll send a tiny new PR to fix it and reference it here.

Reply:
@RX14 @ysbaddaden I fixed this in #6383.